test/sanity/code-smell/docs-build.requirements.in

```diff
@@ -5,4 +5,4 @@ sphinx == 4.2.0
 sphinx-notfound-page
 sphinx-ansible-theme
 straight.plugin
-antsibull
+antsibull-docs
```

Comment:
This is more a future question - I was considering replacing `known_good_reqs.txt` with `test/sanity/code-smell/docs-build.requirements.txt` but if this isn't pinned here, it means that anytime core does a test container update, they could pull in a more recent version of antsibull-docs than what I've tested, right?  I'm not sure whether that's okay, since we do test at PR stage like this, or not okay, because it would change w/o me realizing it sometime in the future?

Reply:
The `*.in` files are not pinned, but they are used to generate the `*.txt` files which are pinned, in this case `test/sanity/code-smell/docs-build.requirements.txt`, which this PR pins to `1.0.0`.